graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/ConstantsGenerator.kt

```diff
@@ -81,23 +81,28 @@ class ConstantsGenerator(
                 types[it.name] = constantsType
             }
 
-        types.values.forEach {
-            javaType.addType(it.build())
-        }
 
         document.definitions
             .filterIsInstance<InputObjectTypeDefinition>()
             .asSequence()
             .excludeSchemaTypeExtension()
             .forEach {
-                val constantsType = createConstantTypeBuilder(config, it.name)
-                constantsType.addField(
-                    FieldSpec
-                        .builder(ClassName.get(String::class.java), "TYPE_NAME")
-                        .addModifiers(Modifier.PUBLIC, Modifier.STATIC, Modifier.FINAL)
-                        .initializer("\$S", it.name)
-                        .build(),
-                )
+                val constantsType =
```

Comment:
shall `KotlinConstantsGenerator` also be updated?

Reply:
Thanks for the reminder, forgot about it being separate. Fixed.